staticaddr/address/manager.go

```diff
@@ -60,6 +56,7 @@ type Manager struct {
 
 // NewManager creates a new address manager.
 func NewManager(cfg *ManagerConfig) *Manager {
+	log = staticaddr.GetLogger()
```

Comment:
Why is this change needed? Isn't `sync.Once` needed here to prevent `log` from being simultaneously overwritten and used for logging?

Reply:
There is another instance in `deposit` package.

Reply:
`address`, `deposit` and `withdraw` all share the same instance of the logger. I think `GetLogger` returns a single instance of the logger from the `init()` call.

How else would you access the common log instance from `staticaddr/log.go`? Pass it into each of the managers?

Reply:
Thank you, I've applied this change.